pinot-query-runtime/src/test/resources/queries/Aggregates.json

```diff
@@ -0,0 +1,295 @@
+{
+  "general_aggregate": {
+    "tables": {
+      "tbl": {
+        "schema": [
+          {
+            "name": "int_col",
+            "type": "INT"
+          },
+          {
+            "name": "double_col",
+            "type": "DOUBLE"
+          },
+          {
+            "name": "string_col",
+            "type": "STRING"
+          }
+        ],
+        "inputs": [
```

Comment:
nit: can't we make the inputs and schemas one line cross? makes it easier to read instead of the auto-formatting

I will create a codestyle XML changes for JSON type from "always wrap" to "wrap if too long"

Reply:
Yeah. I started like that but the auto formatter changed for me. How about I auto format after you add a rule?

Reply:
auto format doesn't seem to be working